src/test/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetStatisticsRDDSuite.scala

```diff
@@ -484,4 +485,13 @@ class ParquetStatisticsRDDSuite extends UnitTestSuite with SparkLocal {
     val hosts = rdd.getPreferredLocations(partition)
     hosts should be (Nil)
   }
+
+  test("ParquetStatisticsRDD - getFS, check file://") {
+    val hadoopConf = spark.sessionState.newHadoopConf()
+    // ideally should have tested s3a here but that needs credentials 
+    // and also AWS SDK classes in classpath
+    hadoopConf.set(ParquetMetastoreSupport.FILTER_DIR, "file://issue-86-test/index_metastore")
+    val fs = ParquetStatisticsRDD.getFS(hadoopConf)
+    fs shouldBe a [FileSystem]
```

Comment:
It will always be a `FileSystem`. I think it would be better to check `fs.getScheme() should be "file"` - I don't quite remember the value for local file system:). 

And another test (or extend this test) is setting the default file system to local, but unset FILTER_DIR config, so that we test another branch of the code.